src/components/new-request/search.vue

```diff
@@ -400,7 +400,9 @@ export default class NewSearch extends Mixins(CommonMixin) {
       return
     }
     if (type && this.getIsConversion) {
-      let { entity_type_cd } = this.$conversionTypes.find(conv => conv.value === type)
+      // convert NrRequestTypeCodes -> EntityType
+      const value = type as unknown as NrRequestTypeCodes
+      let { entity_type_cd } = this.$conversionTypes.find(conv => conv.value === value)
```

Comment:
This is the local setter for `entity_type_cd` (used for v-model props). However, depending on the menu, this value may be from the EntityType enum or from the NrRequestTypeCodes enum. This fixes the comparison error, allowing the code to find the conversion type with this value, extract the entity type code from it, and then set it in the store.